### ISSUE TITLE
Slideshow buttons standardization.

### ISSUE THREAD

**Issue by OqwtVMwTqYGc9QudgZrkQysmjMPY0XKgbA46yLj** [deckerst/aves#1090]
**Is your feature request related to a problem? Please describe.**
In order to delete or share an image while in slideshow, the "Show In Collection" UI button must be pressed, the collection loaded, and then the image may be deleted or shared through regular gallery UI.
This is somewhat inconvenient, and time-inefficient.

**Describe the solution you'd like**
Up to developer, really. Consider simply repeating the normal image option buttons from the basic gallery, tucking "Cast", "Show In Collection", and slideshow "Settings" into the Tri-Dot Context button. This improves intuitivity. The "Play" button can be removed, as it is redundant to tapping the screen. Perhaps replace it with a play/pause indicator.

**Additional context**
Use case: I save images on my phone and then use slideshows to swipe through them randomly. If I come to an image I don't want to see, I'd like to be able to quickly delete it. If I come to an Image I'd like to edit, I'd like to share it to an app that can edit it, and then return to Aves, at the same image, to delete it or further engage it if necessary. "Share" can do alot.

I feel that "Cast", "Show In Collection", and "Settings" are one-time use buttons. If someone is using Slideshow traditionally, to put some inane visual noise on a screen for a while, then they engage those to initialise the slideshow and then move on.
Statisticly I can't be the only weirdo who uses slideshow like this, and "Share" and "delete" can need be used frequently and repeatedly.

Regardless. thank you for your contribution to the world. You've made a competent, powerful, feature rich, easy to use, free and open source gallery app. Thank you. Fuck corpos.

**Comment by deckerst** [comment]
You are right that you are not the only one who wants to access more actions from the slideshow. At the moment the slideshow customization is not compatible with the regular viewing features and actions, but I'd like to remedy that.

Just curious, if you could randomize the viewer image order, would you still use the slideshow as you do today?

> Regardless. thank you for your contribution to the world. You've made a competent, powerful, feature rich, easy to use, free and open source gallery app. Thank you. Fuck corpos.

Thanks for the kind words!

**Comment by OqwtVMwTqYGc9QudgZrkQysmjMPY0XKgbA46yLj** [comment]
>if you could randomize the viewer image order,...
You mean the actual gallery view? That'd be a useful feature, maybe engaged with a shuffle button.
A non-FoSS app called Random Image Gallery, on Google Play, does this. It's neat, but that basic slideshow functionality is sufficient.

**Comment by DroidFloyd** [comment]
Good day Dev. Please prioritize this issue when you can. Slideshow is so bland and unintuitive without more options and multi select capabilities.
